File: 1-İkiSayınınBölünmesi/IkiSayininBolunmesi.java
import java.util.*;
public class IkiSayininBolunmesi {
 public static void main(String arg[]) {
  Scanner input = new Scanner(System.in);
  // “a”, “b”, “c” tanımlanması
  int a, b, c; 

  // “a” ve “b”nin değerlerinin girilmesi
  System.out.print("a=");
  a = input.nextInt(); 
  System.out.print("b=");
  b = input.nextInt(); 
  
  // Büyük sayının a'ya alınması
  if (a < b) 
  {
   c = a; // “a”nın değerinin “c”de saklanması
   a = b; // “b”nin değeri “a”ya atanması
   b = c; // “c”de Saklanan değerinin “b”ye aktarılması
  }
  
  if (a % b == 0) // 0 ise “Tam Bolunebilir” yazdırılması
   System.out.println("Tam Bolunebilir");
  else // 0 değilse “Tam Bolunemez” yazdırılması
   System.out.println("Tam Bolunemez");
 }
}

File: 1-İkiSayınınBölünmesi/IkiSayininBolunmesi.cpp
#include <iostream>

using namespace std;
int main()
{
    // “a”, “b”, “c” tanımlanması
    int a, b, c;
    cout<< "a= ";
  
    // “a” ve “b”nin değerlerinin girilmesi
    cin >> a;
    cout << "b= ";
    cin >> b;

    // Büyük sayının tespit edilmesi.
    if (a < b)
    {
        // “a”nın değerinin “c”de saklanması  
        c = a;
        // “b”nin değeri “a”ya atanması
        a = b;
        // “c”de Saklanan değerinin “b”ye aktarılması
        b = c; 
    }
  
    // 0 ise “Tam Bolunebilir” yazdırılması
    if (a % b == 0) 
        cout << "Tam Bolunebilir";
    // 0 değilse “Tam Bolunemez” yazdırılması
    else 
        cout << "Tam Bolunemez";
    
    return 0;
}
